jitterdebugger.c

```diff
@@ -60,6 +60,8 @@ struct stats {
 	unsigned int avg;	/* in us */
 	unsigned int hist_size;
 	uint64_t *hist;		/* each slot is one us */
+	uint64_t *hist_overflows; /* continous list */
```

Comment:
hist_overflows is not a list. It's array, no? Maybe you could also short had overflows to 'ovf' which is often also found in the kernel as shorthand (this would also reduce the line lenght below :))

Reply:
`hist_overflow` is an array, but since `hist` is also an array (per se a bucket array), describing the former as a list seems more fitting. 
Ack for `_ovf`, will apply.